.changeset/orange-countries-accept.md

```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/performance-compat": patch
```

Comment:
The changeset should be removed. exp and compat packages don't use changeset yet.

Reply:
Done.